runtime/doc/lsp.txt

```diff
@@ -1437,13 +1437,14 @@ type_definition({options})                     *vim.lsp.buf.type_definition()*
     Parameters: ~
       • {options}  (`vim.lsp.LocationOpts?`) See |vim.lsp.LocationOpts|.
 
-typehierarchy({kind})                            *vim.lsp.buf.typehierarchy()*
+typehierarchy({kind}, {options})                 *vim.lsp.buf.typehierarchy()*
```

Comment:
Ah right, I basically `vim.lsp.buf.references()` also calls this `options` instead of `opts`, we should probably change that as well then?

Reply:
> I basically copied this from `vim.lsp.buf.references()` which also calls this `options` instead of `opts`, we should probably change that as well then?

yes. shorter is better. and "options" means something very different in vim/nvim.

Reply:
Right, I added an additional commit that fixes this for all other functions in buf.lua